librz/debug/p/debug_winkd.c

```diff
@@ -268,7 +338,7 @@ RzDebugPlugin rz_debug_plugin_winkd = {
 	.arch = "x86",
 	.bits = RZ_SYS_BITS_32 | RZ_SYS_BITS_64,
 	.init = &rz_debug_winkd_init,
-	.step = &rz_debug_winkd_step,
+	// TODO: .step = &rz_debug_winkd_step,
```

Comment:
Is it unfinished in this PR?

Reply:
It didnt work before because it was never implemented, it just returned true. Now rizin will manually do the work to simulate a step.

Reply:
What does it mean that "it manually do the work"? Where? Why was not implemented in `rz_debug_winkd_step`?

Reply:
@ret2libc It means that rizin will use software stepping (`dbg.swstep`), manually disassemble the current instruction and put a breakpoint in the next instruction to be executed, following jump/rets.